sunpy/tests/__init__.py

```diff
@@ -81,8 +81,13 @@ def main(modulename='', coverage=False, cov_report=False,
         all_args.append('--remote-data')
     if not offline:
         all_args.append('-k remote_data')
+
+    if figure_only:
+        figure = True
     if not figure:
         all_args.append('-m not figure')
+    if figure_only:
+        all_args.append('-m figure')
```

Comment:
Why is the if statement repeated twice?

Reply:
I added one more argument to only select figure.I left the previous case to include figure untouched.

Reply:
Could the two `if figure_only:` statements not be merged?